.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,13 @@
+contact_links:
+    - name: 📃 Admin Panel Issue
+      url: https://github.com/PalisadoesFoundation/talawa-admin/issues/new/choose
+      about: This issue tracker is not for Admin Panel issues. Please file Admin Panel issues here.
+    - name: 📃 Mobile Issue
```

Comment:
Remove all the links to creating issues in other repositories. They are not relevant